website/widgets/form/table.md

```diff
@@ -58,19 +58,18 @@ q.page['example'] = ui.form_card(box='1 1 5 3', items=[
 
 ## With selection
 
-If `multiple` is set to False (default), each row in the table is clickable. When a cell in the column with `link=True`
+By default, each row in the table is clickable. When a cell in the column with `link=True`
 (defaults to first column) is clicked or the row is doubleclicked, the form is
 submitted automatically and `q.args.table_name` is set to `[row_name]`, where
 `table_name` is the `name` of the table, and `row_name` is the `name` of the row that was clicked.
 Note that doubleclicking the row if single selection is active results in submission as well.
```

Comment:
This probably needs to be changed as well if I read the code correctly. Double-clicking works only if the selection mode is none, right?

Reply:
Right. Especially this sentence is confusing:

> Note that doubleclicking the row if single selection is active results in submission as well.

What did you mean by this when there was not single selection supported before?

Reply:
> What did you mean by this when there was not single selection supported before?

Previously, single selection meant no checkmark, just direct submission.